test_util/cluster.py

```diff
@@ -323,15 +323,14 @@ def install_dcos(
 def upgrade_dcos(cluster, installer_url, add_config_path=None):
 
     def upgrade_host(tunnel, role, bootstrap_url):
-        # Download the install script for the new DC/OS.
-        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/dcos_install.sh'])
+        # Download the upgrade script for the new DC/OS.
+        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/upgrade.latest'])
+        upgrade_id = tunnel.remote_cmd(['cat', 'upgrade.latest'])
```

Comment:
Going forward do we test our old way of upgrades or just do this one? I need to either add a new test , currently I am modifying the one that existed.